From 2901f1cb2b8b27fe5d3aba0a318c8487c8ea6698 Mon Sep 17 00:00:00 2001
From: Chris Burr <christopher.burr@cern.ch>
Date: Mon, 4 Dec 2023 05:57:02 +0100
Subject: [PATCH] Fix TestProcess.cc for boost 1.81

---
 Tests/Core/TestProcess.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Tests/Core/TestProcess.cc b/Tests/Core/TestProcess.cc
index a18a01f0..33c94e99 100644
--- a/Tests/Core/TestProcess.cc
+++ b/Tests/Core/TestProcess.cc
@@ -85,8 +85,8 @@ namespace Hammer {
         auto tmpT = combineDaughters({-16, 14, -13}, {});
         HashId idT = processID(-15, tmpT);
         HashId seed = 0ul;
-        boost::hash_combine(seed,idB);
         boost::hash_combine(seed,idT);
+        boost::hash_combine(seed,idB);
 
         EXPECT_EQ(proc.getId(),seed);
 
-- 
2.34.1

